cfme/utils/template/rhevm.py

```diff
@@ -35,11 +35,9 @@ def import_template_from_glance(self):
         self.mgmt.import_glance_image(
             source_storage_domain_name=self.glance_key,
             target_cluster_name=self.provider_data.template_upload.cluster,
-            # This part depends on wrapanapi PR + release
-            # https://github.com/ManageIQ/wrapanapi/pull/264
             source_template_name=self.image_name,
             target_template_name=self.temp_template_name,
-            target_storage_domain_name=self.provider_data.template_upload.sdomain)
+            target_storage_domain_name=self.provider_data.template_upload.storage_domain)
```

Comment:
why this rename? `sdomain` <> `storage_domain` just readability or something else?

Reply:
readability